src/ch/imedias/rsccfx/view/HeaderView.java

```diff
@@ -18,12 +18,14 @@
  */
 public class HeaderView extends HBox {
   private static final double HEADER_HEIGHT = 250d;
+  private static final double WIDTH_HEIGHT_PROPERTY_VALUE = 50d;
```

Comment:
I think this name is not descriptive enough, it's not clear which width and height it is for. From what I see from the code, it seems like it is the width and height of each button? In this case I'd suggest: "BUTTON_WIDTH_HEIGHT" or maybe even "BUTTON_SIZE"?